mlir/tools/mlir-miopen-driver/mlir-miopen-driver.cpp

```diff
@@ -290,6 +309,33 @@ static void correctParameters() {
     else
       inputLayout.setValue("g" + inputLayoutValue);
   }
+
+  if (paddingHeight.getValue() > 0) {
+    if (paddingHeightLeft.getValue() == 0 &&
```

Comment:
better to leave comment here to specify `padding_h` supersedes `padding_h_l` and `padding_h_r`. same goes with `padding_w`.